src/main/java/org/spongepowered/api/entity/EntityTypes.java

```diff
@@ -372,12 +368,7 @@
 
     public static final DefaultedRegistryReference<EntityType<ZombifiedPiglin>> ZOMBIFIED_PIGLIN = EntityTypes.key(ResourceKey.minecraft("zombified_piglin"));
 
-    // SORTFIELDS:OFF
-
     // @formatter:on
-
-    public static final DefaultedRegistryReference<EntityType<Human>> HUMAN = EntityTypes.key(ResourceKey.sponge("human"));
```

Comment:
:(

Reply:
whee time to add a new feature to the generator